app/rdf/query-geo-shapes.ts

```diff
@@ -62,110 +60,105 @@ export const createGeoShapesLoader =
           ?iri ${ns.schema.name} ?label .
         }`.prologue`${pragmas}`;
 
-      let rawLabels: RawLabel[] = [];
+    let rawLabels: RawLabel[] = [];
 
-      try {
-        rawLabels = (await labelsQuery.execute(sparqlClient.query, {
-          operation: "postUrlencoded",
-        })) as RawLabel[];
-      } catch (e) {
-        console.error(e);
-      }
+    try {
+      rawLabels = (await labelsQuery.execute(sparqlClient.query, {
+        operation: "postUrlencoded",
+      })) as RawLabel[];
+    } catch (e) {
+      console.error(e);
+    }
+
+    const groupedLabels = groupBy(
+      rawLabels.map((d) => {
+        return {
+          iri: d.iri.value,
+          label: d.label?.value ?? d.iri,
+        };
+      }),
+      (d) => d.iri
+    );
 
-      const groupedLabels = groupBy(
-        rawLabels.map((d) => {
-          return {
-            iri: d.iri.value,
-            label: d.label?.value ?? d.iri,
-          };
-        }),
-        (d) => d.iri
-      );
-
-      const geometriesQuery = SELECT`?iri ?geometry`.WHERE`
+    const geometriesQuery = SELECT`?iri ?geometry`.WHERE`
         VALUES ?iri {
           ${queryableIris}
         }
 
         ?iri ${ns.geo.hasGeometry} ?geometry .`.prologue`${pragmas}`;
 
-      let rawGeometries: RawGeometry[] = [];
+    let rawGeometries: RawGeometry[] = [];
 
-      try {
-        rawGeometries = (await geometriesQuery.execute(sparqlClient.query, {
-          operation: "postUrlencoded",
-        })) as RawGeometry[];
-      } catch (e) {
-        console.error(e);
-      }
+    try {
+      rawGeometries = (await geometriesQuery.execute(sparqlClient.query, {
+        operation: "postUrlencoded",
+      })) as RawGeometry[];
+    } catch (e) {
+      console.error(e);
```

Comment:
Nit: this try/catch can be a bit simplified by using catch() directly. This way you do not have to declare rawGeometries before hand.

```
const rawGeometries = await stuff().catch(e => {
        console.error(e)
	return []
})
```